socket/src/main/java/net/runelite/client/plugins/socket/plugins/sotetseg/SotetsegPlugin.java

```diff
@@ -0,0 +1,337 @@
+/*
```

Comment:
this should be added to the Theatre of blood plugin, not bundled into the socket plugin.

Reply:
How does this work? Make TOB plugin depend on Socket as well?